### DISCUSSION TITLE
Combine Multiple Samples

### DISCUSSION THREAD

**Discussion by marsdenl** [giotto-suite/Giotto#973] [General]
Hi there, 

I'm very new to Giotto and general spatial transcriptomics. Having done Seurat single-cell analyses before, I know multiple samples can be combined into a single annotated seurat object for DEG analysis. I was wondering if there was a similar framework to combine multiple samples (different mice, same area) for post-hoc analyses in Giotto. The hope is to make sure spatial domains or spatially enriched genes are consistent across those samples. Thanks for your help!

**Comment by josschavezf** [comment]
Hi, you can put multiple samples in the Giotto object using the function joinGiottoObjects, here is an example pipeline https://drieslab.github.io/Giotto_website/articles/visium_prostate_integration.html

**Comment by marsdenl** [comment]
Hey @josschavezf! Apologies for the delayed response on my end. Thank you for your answer, helps a lot :)
